iOS-EN（裁剪版）/headers/SDK/WhiteSDK.h

```diff
@@ -137,6 +137,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)setCommonCallbackDelegate:(nullable id<WhiteCommonCallbackDelegate>)callbackDelegate;
 
+/**
+ Writes the slide log into a specified file.
+ 
+ @param path The path to the specified file. If the file does not exist, it will be created. If the file already exists, the slide log will be written at the end of the file. 
```

Comment:
Edit: "it is created." (avoid future tense)
Edit: "is written" (or "is appended to the end of the file.")

Reply:
Wow, I like "append to". thx~